### PR TITLE
feat: Secondary nav items

### PR DESCRIPTION
**Pull request by ptbrowne**
Added secondary nav items.

Extracted styles from store, for usage in Banks
Took this oppportunity to make our example a bit more interactive and closer to reality.

https://ptbrowne.github.io/cozy-ui/react/#/Layout

### CODE REVIEW

File: react/Layout/Layout.md [hunk 1]
```diff
@@ -2,51 +2,90 @@ The Layout component brings a strong context for apps with any screen resolution
 
 * `<Main />` is considered the wrapper of your app content. `<Content />` should be a direct children of it but you can add other components as well, like a toolbar and such.
 * `<Content />` is the main content or your app.
+* Secondary `<NavItem />` are not visible on mobile
```
Comment (Crash--) [inline react/Layout/Layout.md:5]:
```suggestion
⚠️  Secondary `<NavItem />` are not visible on mobile
```
 ?

Comment (Crash--) [reply react/Layout/Layout.md:5]: Also, we display none on +medium-screen. We should tell so, this is not just on mobile right? Tablets are also concerned?

File: stylus/components/nav.styl [hunk 2]
```diff
@@ -126,3 +130,45 @@ $nav-link
...
             color var(--slateGrey)
+
+
+$nav-item-secondary
+    height auto
+    +medium-screen()
+        &
+            display none
+
+    &
```
Comment (y-lohse) [inline stylus/components/nav.styl:141]: We can simplify this blog by removing the ampersands and moving the margin-top declaration.

File: react/Layout/Layout.md (hunk 1 above)
Comment (ptbrowne) [reply react/Layout/Layout.md]: Yes tablets are also concerned, I updated the doc to make it clear and added the warning icon.

File: stylus/components/nav.styl (hunk 2 above)
Comment (ptbrowne) [reply stylus/components/nav.styl]: Done

**Pull request merged**